src/View/batchprocessing/ROINameCleaningOptions.py

```diff
@@ -318,16 +319,29 @@ def populate_table(self, dicom_structure, batch_directory):
             # Loop through each ROI in the RT Struct
             for i in range(len(rtstruct.StructureSetROISequence)):
                 # Get the ROI name
-                roi_name = rtstruct.StructureSetROISequence[i].ROIName
+                roi_name = rtstruct.StructureSetROISequence[i].ROIName.lstrip()
 
                 # Add ROI name to the dictionary
                 if roi_name not in rois.keys():
                     rois[roi_name] = []
+                
+                # Get the suffix of the ROI name if it is a integer
+                temp = re.search(r'\d+$', roi_name)
```

Comment:
could use ```if roi_name.isdigit():``` here? regex might not be necessary

Reply:
I was taking out the back of the string and getting the numbers i.e. PTV1111 I would get 1111 so I could see if the length of the numbers were equal to 4. Now thinking about this I could write a couple more lines so that regex wouldn't be necessary such as roi_name[4:8].isdigit() != True and roi_name[3:7] != True respective of what prefix I'm checking in the if statement below. Interested in your thoughts

Reply:
Ah okay what you did is great then. I saw the $ sign in the regex and was assuming it was a single number you were searching eg. '1111'. Apologies, I should have thought about the fact that you had clearly written SUFFIX  in the commensts! good job